.circleci/cimodel/data/windows_build_definitions.py

```diff
@@ -132,10 +132,10 @@ def TruePred(_):
     WindowsJob(None, _VC2019, CudaVersion(10, 1)),
     WindowsJob(1, _VC2019, CudaVersion(10, 1)),
     WindowsJob(2, _VC2019, CudaVersion(10, 1)),
-    # VS2019 CUDA-11.1
-    WindowsJob(None, _VC2019, CudaVersion(11, 1)),
-    WindowsJob(1, _VC2019, CudaVersion(11, 1), master_only_pred=TruePred),
-    WindowsJob(2, _VC2019, CudaVersion(11, 1), master_only_pred=TruePred),
+    # VS2019 CUDA-11.2
+    WindowsJob(None, _VC2019, CudaVersion(11, 2)),
+    WindowsJob(1, _VC2019, CudaVersion(11, 2), master_only_pred=TruePred),
```

Comment:
Please remove master_only_pred=TruePred for windows cuda11.2 first.
If it passed, then restore master_only_pred.

And please paste the link of  windows cuda11.2 test in the PR.